server/src/modules/rules/constants/rules.constants.ts

```diff
@@ -372,7 +413,7 @@ export class RuleConstants {
           name: 'tags',
           humanName: '[list] Tags (Text if 1, otherwise list)',
```

Comment:
```suggestion
          humanName: '[list] Tags',
```

This seems to always return a list even if there's 1 result, may have been a functionality change in the past.